packages/tools/kadena-cli/src/tx/utils/txSignWithAliasFile.ts

```diff
@@ -113,78 +112,70 @@ export const signTransactionFileWithAliasFile = async (data: {
-    globalOptions.legacy({ isOptional: true, disableQuestion: true }),
-  ],
-  async (option, values, stdin) => {
-    const wallet = await option.keyWallet();
-    if (!wallet.keyWalletConfig) throw new Error('wallet not found');
-
-    const password = await option.securityPassword();
-    const key = await option.keyAliasSelect({
-      wallet: wallet.keyWallet,
-    });
-    const mode = await option.legacy();
-
-    const result = await (async () => {
-      if (stdin !== undefined) {
-        const command = await parseTransactionsFromStdin(stdin);
-        log.debug('sign-with-key-alias-file:action', {
-          ...wallet,
-          ...key,
-          ...mode,
-          command,
-        });
-        return await signTransactionWithAliasFile({
-          alias: key.keyAliasSelect,
-          signed: false,
-          wallet: wallet.keyWalletConfig!,
-          password: password.securityPassword,
-          commands: [command],
-          legacy: mode.legacy,
-        });
-      } else {
-        const { directory } = await option.directory();
-        const files = await option.txUnsignedTransactionFiles({
-          signed: false,
-          path: directory,
-        });
-        const absolutePaths = files.txUnsignedTransactionFiles.map((file) =>
-          path.resolve(path.join(directory, file)),
-        );
-        log.debug('sign-with-key-alias-file:action', {
-          ...wallet,
-          ...key,
-          ...files,
-          directory,
-          ...mode,
-        });
-        return await signTransactionFileWithAliasFile({
-          alias: key.keyAliasSelect,
-          signed: false,
-          wallet: wallet.keyWalletConfig!,
-          password: password.securityPassword,
-          files: absolutePaths,
-          legacy: mode.legacy,
-        });
-      }
-    })();
-
-    assertCommandError(result);
-
-    if (result.data.commands.length === 1) {
-      log.output(JSON.stringify(result.data.commands[0], null, 2));
+export async function signWithAliasFile(
+  option: CommandOption<typeof options>,
+  values: string[],
+  stdin?: string,
+): Promise<void> {
+  const wallet = await option.walletName();
+  if (!wallet.walletNameConfig) throw new Error('wallet not found');
+
+  const password = await option.securityPassword();
+  const key = await option.keyAliasSelect({
+    wallet: wallet.walletName,
+  });
+  const mode = await option.legacy();
+
+  const result = await (async () => {
+    if (stdin !== undefined) {
+      const command = await parseTransactionsFromStdin(stdin);
+      log.debug('sign-with-key-alias-file:action', {
+        ...wallet,
+        ...key,
+        ...mode,
+        command,
+      });
+      return await signTransactionWithAliasFile({
+        alias: key.keyAliasSelect,
+        signed: false,
+        wallet: wallet.walletNameConfig!,
+        password: password.securityPassword,
+        commands: [command],
+        legacy: mode.legacy,
+      });
+    } else {
```

Comment:
You don't need this `else` as we have a return in the previous `if` block

Reply:
fair enoguh